app/handlers/identities_register.rb

```diff
@@ -35,6 +39,12 @@ def handle
       user = outputs[[:create_user, :user]]
     end
 
+    if register_params.email.blank?
+      fatal_error(code: :missing_email,
+                  message: 'Email address is mandatory.')
+    end
```

Comment:
Pretty sure you can replace this block with 

```ruby
validates :email, presence: true, message: "is mandatory."
```

in the `paramify` block up top.